src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -33,7 +35,18 @@ export class RadioButton extends Widget {
 		this.label = opts.label;
 		this.enabled = opts.enabled || true;
 		this.checked = opts.checked || false;
-		this.onclick(this.inputElement, () => this._onClicked.fire());
+		this.onclick(this.inputElement, () => {
+			this._onClicked.fire();
+			let buttonGroup = document.getElementsByName(this.name);
```

Comment:
What's the behavior if name is empty/undefined? We should only do this if we actually have a name